scripts/create_fauxalysis_subset_from_yaml.py

```diff
@@ -0,0 +1,84 @@
+import sys, os, argparse, yaml, shutil
```

Comment:
I think it would be helpful to have some kind of docstring here or in the functions to explain what this script does and how it is supposed to be used.